### ISSUE TITLE
Gray screen with deep zoom

### ISSUE THREAD

**Issue by munrocket** [cslarsen/mandelbrot-js#5]
For now you have limit in zooming (~10^15) because you use javascript 64-bit float. But you can overcome the limit by using floating point expansion technique that is pretty fast. For example you can use [double.js](https://github.com/munrocket/double.js) or [bigfloat](https://github.com/charto/bigfloat) library.

**Comment by cslarsen** [comment]
That is true, but I don't have time to fix it myself. I tried something like that, but was not satisfied with the speed. Perhaps change to arbitrary precision when we reach a certain limit or something.
